website/scripts/website-build.sh

```diff
@@ -0,0 +1,32 @@
+# Repo which we are cloning and executing npm run build:deploy-preview within
+REPO_TO_CLONE=terraform-website
+# Set the subdirectory name for the terraform-website app
+PREVIEW_DIR=website-preview
+CLONE_DIR=website-preview
+
+from_cache=false
+
+if [ -d "$PREVIEW_DIR" ]; then
+    echo "$PREVIEW_DIR found"
+    CLONE_DIR="$PREVIEW_DIR-tmp"
+    from_cache=true
+fi
+
+# Clone the terraform-website project, if needed
+echo "⏳ Cloning the terraform-website repo, this might take a while..."
+git clone --depth=1 --branch=kevin/terraform-split https://github.com/hashicorp/$REPO_TO_CLONE.git "$CLONE_DIR"
```

Comment:
Apply this change when hashicorp/terraform-website#2228 is merged

```suggestion
git clone --depth=1 --branch=master https://github.com/hashicorp/$REPO_TO_CLONE.git "$CLONE_DIR"
```